payment-component/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
@cb-amalgeorge, can we delete this file? It is redundant, correct?

Reply:
This is the html file, which is the page that loads the js code, and defines the DOM. This is a required file.

Reply:
But isn't that the same as `payment-component/vanilla-js/apps/client/index.html`?

Reply:
@cb-amalgeorge perhaps you mean it's needed for the Next.js app?